src/main/java/com/hazelcast/commandline/member/MemberCommandLine.java

```diff
@@ -0,0 +1,341 @@
+
+        Integer pid = buildJavaProcess(HazelcastMember.class, args, foreground, additionalClassPath);
+        saveProcess(new HazelcastProcess(processUniqueId, pid));
+
+        println(processUniqueId);
+    }
+
+    private boolean isNullOrEmpty(String string) {
+        return string == null || string.isEmpty();
+    }
+
+    private String createProcessDirs(String processUniqueId) {
+        String processPath = HAZELCAST_HOME + SEPARATOR + processUniqueId;
+        String logPath = processPath + SEPARATOR + logsDirString;
+        try {
+            new File(logPath).mkdirs();
+        } catch (Exception e) {
+            throw new HazelcastException("Process directories couldn't created.");
+        }
+        return processPath;
+    }
+
+    private String createLoggingPropertiesFile(String processUniqueId, String processDir)
+            throws FileNotFoundException {
+        String loggingPropertiesPath = processDir + SEPARATOR + "logging.properties";
+        PrintWriter printWriter = new PrintWriter(loggingPropertiesPath);
+        String fileContent = "handlers= java.util.logging.FileHandler, java.util.logging.ConsoleHandler\n" +
+                ".level= INFO\n" +
+                "java.util.logging.FileHandler.pattern = " +
+                processDir + SEPARATOR + logsDirString + SEPARATOR + "hazelcast.log\n" +
+                "java.util.logging.FileHandler.limit = 50000\n" +
+                "java.util.logging.FileHandler.count = 1\n" +
+                "java.util.logging.FileHandler.maxLocks = 100\n" +
+                "java.util.logging.FileHandler.formatter = java.util.logging.SimpleFormatter\n" +
+                "java.util.logging.FileHandler.append=true\n" +
+                "java.util.logging.ConsoleHandler.formatter = java.util.logging.SimpleFormatter";
+        printWriter.println(fileContent);
+        printWriter.close();
+        return loggingPropertiesPath;
+    }
+
+    @Command(description = "Stops a Hazelcast IMDG member",
+            mixinStandardHelpOptions = true
+    )
+    public void stop(
+            @Parameters(index = "0",
+                        paramLabel = "<process-id>",
+                        description = "Process id of the process to stop, for ex.: brave_frog")
+                        String processUniqueId) throws IOException, ClassNotFoundException {
+        HazelcastProcess process = getProcessMap().get(processUniqueId);
+        if (process == null){
+            printlnErr("No process found with process id: " + processUniqueId);
+            return;
+        }
+        int pid = process.getPid();
+        Runtime.getRuntime().exec("kill -15 " + pid);
+        removeProcess(processUniqueId);
+        println(processUniqueId + " stopped.");
+    }
+
+    @Command(description = "Lists running Hazelcast IMDG members",
+            mixinStandardHelpOptions = true
+    )
+    public void list() throws ClassNotFoundException {
```

Comment:
Yes, it is an issue. Because of that, I am planning to add a status information for members, such as `RUNNING`, `STOPPED`, etc. I prefer to do it in another PR, if you are also ok with it. I will create a GH issue for this problem.